server/sonar-web/src/main/js/apps/permission-templates/groups-view.js

```diff
@@ -0,0 +1,52 @@
+import Modal from '../../components/common/modals';
+import '../../components/common/select-list';
+import './templates';
+
+function getSearchUrl (permission, permissionTemplate) {
+  return baseUrl + '/api/permissions/template_groups?ps=100&permission=' + permission +
+      '&templateId=' + permissionTemplate.id;
+}
+
+export default Modal.extend({
+  template: Templates['permission-templates-groups'],
+
+  onRender: function () {
+    this._super();
+    new window.SelectList({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace by a standard call to the function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AConstructorFunctionsForSideEffects)